ccaas_builder/cmd/build/main.go

```diff
@@ -0,0 +1,194 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+	"text/template"
+
+	"github.com/otiai10/copy"
+	"github.com/pkg/errors"
+)
+
+var logger = log.New(os.Stderr, "", 0)
+
+type chaincodeMetadata struct {
+	Type string `json:"type"`
+}
+
+// Connection structure is used to represent the
+// connection.json file that is supplied in the
+// chaincode package.
+type connection struct {
+	Address     string `json:"address"`
```

Comment:
These can all be lowercase (not exported).

Reply:
Used to unmarshall JSON data into so the elements need to be exported